services/frontend-service/src/ui/components/ReleaseCard/ReleaseCard.tsx

```diff
@@ -74,8 +74,12 @@ const rolloutStatusPriority = [
 const calculateDeploymentStatus = (
     app: string,
     deployedAt: EnvironmentGroupExtended[],
+    rolloutEnabled: boolean,
     rolloutStatus: RolloutStatusApplication
 ): [Array<StreamStatusResponse>, StreamStatusResponse?] => {
+    if (!rolloutEnabled) {
+        return [[], undefined];
```

Comment:
Ideally we could show a tooltip here that explains the situation. e.g.
`<div title='To show the rollout status, enable the rollout service (...)'>not configured</div>`
(somewhere in the releaseCard)

Reply:
but I'm fine to merge as it is

Reply:
I'm not sure if that's a good idea to put this very prominently. Maybe have an info pane somewhere that shows this along with the versions of the backend services?